frappe_private_comment/overrides/whitelist/comment.py

```diff
@@ -82,3 +84,40 @@ def get_comment_visibility(name: str):
         return None
 
     return {"custom_visibility": doc.custom_visibility}
+
+
+@frappe.whitelist()
+def get_comment_replies(reference_doctype: str, comment_id: str):
+    """Get comment replies"""
+    return frappe.get_all(
+        "Comment",
+        filters={
+            "reference_doctype": reference_doctype,
+            "custom_reply_to": comment_id,
+        },
+        fields=["name", "content", "comment_by", "creation"],
+    )
+
+
+@frappe.whitelist()
+def get_all_replies(reference_doctype: str, reference_name: str):
```

Comment:
`get_all_replies` and `get_comment_replies` can be merged into one endpoint only 🤔

Reply:
Both are doing different things, one is getting all the replies _for a parent Doctype (like ToDo)_, another is getting all the replies for _a parent comment_. Both are returning a response differently (one a dict, another a list of comments)